enhancements/network/ingress-nodeport-publishing.md

```diff
@@ -109,24 +110,25 @@ spec:
     ingresscontroller.operator.openshift.io/deployment-ingresscontroller: default
 ```
 
-The Ingress Operator will ignore any updates to `.spec.ports[].nodePort` fields of the Service.
+**Important**: The Ingress Operator will ignore any updates to `.spec.ports[].nodePort` fields of the Service.
 
-### User Stories [optional]
-
-Detail the things that people will be able to do if this is implemented.
-Include as much detail as possible so that people can understand the "how" of
-the system. The goal here is to make this feel real for users without getting
-bogged down.
+### User Stories
 
 #### Story 1
 
-#### Story 2
+As an OpenShift administrator, I want to integrate the default ingresscontroller with a self-managed load balancer directly through node ports in a way that maximizes ingress utilization and ingresscontroller scheduling flexibility.
+
+### Implementation Details/Notes/Constraints
+
+One critical architectural detail of this proposal which demands scrutiny is the following constraint:
+
+* The Ingress Operator will ignore any updates to `.spec.ports[].nodePort` fields of the Service.
 
-### Implementation Details/Notes/Constraints [optional]
+By making explicit that users own the  `.spec.ports[].nodePort` field, no additional port configuration API should be required. By default, ports are allocated automatically, and if those automatically allocated ports are insufficient, users can update the managed `Service` resource directly.
```

Comment:
What does "insufficient" mean?  Does this refer to the use-case in which the user determines the ports ahead of time (possibly as part of firewall configuration) and then configure the ingresscontroller to use those predetermined ports?

Reply:
I've tried to clarify this use case.